abnormal_security/help.md

```diff
@@ -0,0 +1,231 @@
+# Description
+
+Protect your Microsoft Office 365 and G-Suite environments with next-generation email security that uses the most advanced AI detection techniques to stop targeted phishing attacks
+
+# Key Features
+
+* Retrive a list of threats from Abnormal Threat Log
+* Get details about existing threats
+
+# Requirements
+
+* Example: Requires an API Key from the product
```

Comment:
Is there a link to docs that show how to generate a key that we can add below: []()

```suggestion
An Abnormal Security API Key
```

Reply:
Unfortunately, I was unable to find any link to Abnormal Security documentation 🙁